core/src/Streamly/Internal/Data/Fold/Type.hs

```diff
@@ -416,12 +416,11 @@ import Prelude hiding (concatMap, filter, foldr, map, take)
 -- >>> import Streamly.Data.Fold (Fold)
 -- >>> import Streamly.Internal.Data.Stream.Type (Stream)
 -- >>> import qualified Data.Foldable as Foldable
--- >>> import qualified Streamly.Prelude as Stream
+-- >>> import qualified Streamly.Prelude as PStream
```

Comment:
The convention is to import it as IsStream, that way we can also find it easily to replace later.